### ISSUE TITLE
example_usage/config.obs failed 

### ISSUE THREAD

**Issue by tkleiner** [ismip/ismip6-antarctic-ocean-forcing#22]
I followed the instructions in `example_usage/README.md`, but 
```
conda create -n ismip6_ocean_forcing -c conda-forge python=3.6 numpy scipy \
    matplotlib netCDF4 xarray progressbar2 basemap descartes cartopy shapely \
    nco gsw scikit-fmm phshp
```
failed with the following error message
```
Fetching package metadata .................

PackageNotFoundError: Packages missing in current channels:
            
  - phshp
...
```
I decided to try without `phshp`, because I only found it in `setup.py`(line 29).
```
./setup.py:29:                        'shapely', 'gsw', 'scikit-fmm', ' phshp'],
```
BTW: Should this be `'phshp' `instead of `' phshp'`?

Unfortunately 
`python -m ismip6_ocean_forcing config.obs` failed with:
```
...

  Extending IMBIE basins beyond ice fronts...
    A-Ap
Traceback (most recent call last):
  File "/home/ollie/tkleiner/.conda/envs/ismip6_ocean_forcing/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/ollie/tkleiner/.conda/envs/ismip6_ocean_forcing/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/work/ollie/tkleiner/ISMIP6-Projections-Antarctica/pre-processing/work_dir_cosmos/ismip6_ocean_forcing/__main__.py", line 61, in <module>
    main()
  File "/work/ollie/tkleiner/ISMIP6-Projections-Antarctica/pre-processing/work_dir_cosmos/ismip6_ocean_forcing/__main__.py", line 54, in main
    make_imbie_masks(config)
  File "/work/ollie/tkleiner/ISMIP6-Projections-Antarctica/pre-processing/work_dir_cosmos/ismip6_ocean_forcing/imbie/make_imbie_masks.py", line 50, in make_imbie_masks
    extend.extend_imbie_masks(res, basins, bedFileName)
  File "/work/ollie/tkleiner/ISMIP6-Projections-Antarctica/pre-processing/work_dir_cosmos/ismip6_ocean_forcing/imbie/extend.py", line 27, in extend_imbie_masks
    image = misc.imread(imageFileName)
AttributeError: module 'scipy.misc' has no attribute 'imread'
```

Scipy version 1.3.1 was installed via conda. According to [stackoverflow.com](https://stackoverflow.com/questions/15345790/scipy-misc-module-has-no-attribute-imread) `scipy.misc.imread` is not available in since version 1.2.0. 
This worked for me:
```
$ conda uninstall -n ismip6_ocean_forcing -c conda-forge scipy
$ conda install -n ismip6_ocean_forcing -c conda-forge python=3.6 scipy=1.1.0 cartopy
```
Several other options are given in the stackoverflow posts, but all of them would require to touch the code.

**Comment by xylar** [comment]
@tkleiner, thanks for bringing both of these issues to my attention.  The first one should be fixed in #21.  It should have been `pyshp`.

I should be able to fix the second issue by switching to `imageio` in that same pull request.  Thanks for the StackOverflow link.

I'm afraid I don't usually have time to test the full code each time I process new data sets so some features tend to atrophy over time.   I'm aware of depricated features I'm using from `xarray` that will need attention soon but I don't currently have time to figure out the right way to handle those cases.  Just a heads up that `xarray.open_mfdataset` may also cause you trouble in future versions of xarray.

**Comment by tkleiner** [comment]
Thank you for your efforts to keep this running and usable.
